adrian/script.js

```diff
@@ -0,0 +1,46 @@
+//1
+let board = [
+  [1, 2, 3],
+  ["quick", "brown", "fox", "jumped", "over", "lazy", "dog"],
+  [true, false],
+];
+// board.map((array) => {
+//   return array.map((item) => console.log(item));
+// });
+
+//2
+//2.1
+const doggo = {
+  name: "dog",
+  breed: "some",
+  //2.3
+  favoriteFoods: ["tomatoes", "cucumber", "flies"],
+
+  //2.4
+  printFood: function () {
+    this.favoriteFoods.map((food) => console.log(food));
+  },
+};
+//2.3
+let secondElement = doggo.favoriteFoods[1];
+
```

Comment:
Please call the function to print all the doggo's favorite food. 
  doggo.printFood()

Reply:
I did it, but the exercise doesn't say I have to call it, it says: "2.4 Add a method that loops through and print all the doggo's favorite food." That is why I didn't call it.